spec/Vehicle/Connectivity.vspec

```diff
@@ -21,3 +21,130 @@ IsConnectivityAvailable:
   comment: This signal can be used by onboard vehicle services to decide what features that
            shall be offered to the driver, for example disable the 'check for update' button if vehicle does
            not have connectivity.
+           
+ConnectivityType:
+  datatype: string[]
+  type: attribute
+  allowed: [
+    'CELLULAR',
+    'WIFI'
+    ]
+  description: Type of connectivity currently available.
+  
+CurrentDownload:
+  datatype: float
+  type: sensor
+  unit: MiB/s
+  description: Download bandwidth currently in-use.
+
+CurrentUpload:
+  datatype: float
+  type: sensor
+  unit: MiB/s
+  description: Upload bandwidth currently in-use.
```

Comment:
There is an overall CurrentDownload and CurrentUpload, with the understanding that this would simply correspond to whatever ConnectivityType is in-use. This may not work if for some reason cellular and WiFi were blended, but this scenario is relatively complex so I wasn't sure if it was worth considering over a more simplistic model.